src/Search.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Products from './Products';
+
+class _Search extends React.Component{
+    constructor(){
+        super()
+        this.state =  {
+            searchItems: [],
+            inpu: ''
```

Comment:
spelling error on line 10. should be 'input'

Reply:
fixed